templates/etc_systemd_system_postgresql.service.d_custom.conf.j2

```diff
@@ -0,0 +1,11 @@
+# {{ ansible_managed }}
+# Systemd unit file override to specify user/group as well as separate config
+# and data directories.
+[Service]
+User={{ postgresql_service_user }}
+Group={{ postgresql_service_group }}
+
+Environment=PGDATA={{ postgresql_conf_directory }}
+
+ExecStartPre=
```

Comment:
Yes, it clears the ExecStartPre directive for the unit before setting a new one. Without that line, the ExecStartPre directive below this line would be appended to the existing directive, which does not produce the desired effect of checking the data directory instead of PGDATA.